flink-kubernetes/src/main/java/org/apache/flink/kubernetes/cli/KubernetesCliOptions.java

```diff
@@ -46,15 +44,16 @@ public static Option getOptionWithPrefix(Option option, String shortPrefix, Stri
 		.longOpt("clusterId")
 		.required(false)
 		.hasArg(true)
-		.desc(KubernetesConfigOptions.CLUSTER_ID.description().toString())
+		.desc("The cluster id that will be used for flink cluster. If it's not set, the client will generate " +
```

Comment:
nit: remove `that`, 
....`used for identifying the unique flink cluster`?

Reply:
Make sense to me.